examples/30_extended/datasets_tutorial.py

```diff
@@ -117,31 +117,31 @@
 # This example uses the test server, to avoid editing a dataset on the main server.
 openml.config.start_using_configuration_for_example()
 ############################################################################
-# Changes to these field edits existing version: allowed only for dataset owner
+# Change the non-critical fields
```

Comment:
Could you please give more details here, like what are critical fields, what fields can be edited by whom etc?